src/components/JobWizard/JobWizardSteps/JobWizardFunctionSelection/JobWizardFunctionSelection.js

```diff
@@ -306,6 +310,7 @@ const JobWizardFunctionSelection = ({
   ])
 
   const selectProjectFunction = functionData => {
+    selectedActiveTab.current = activeTab
```

Comment:
it's wrong place for setting the active tab because we can have confirm dialog and cancel selecting the new card. It must be in the confirm handler

Reply:
done.